documentation/how-to/protect-api.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Protect an API
+---
+
+This guide explains how to add protection to your GatewayService, by utilizing supported plugins.
+
+Protecting your API is crucial for several reasons:
```

Comment:
On further consideration, I think this page would be best as a concept page, introducing the overall concept of API protection, as well as outlining the different options and their use cases. Then the content on _how-to_ use them can be provided in separate how-to guides (like the one we have for client cred).

The concept page should include all of the available API protection methods we offer (or even one's we don't offer if there was something common that ppl want to do but we say 'sorry can't do that'), and we can flesh out the use cases / pros/cons and how-to's over time.

I have left my original comment below.

---

This intro reads a lot like a ChatGPT generated list... if it isn't that, well my apologies. That said, it's fundamentally decent content but I think it can/should probably be tightened up for the context - we don't need such a lengthy intro in a how-to.

From template:
```
How-to pages have minimal explanation, but often provide links
to conceptual topics that provide related background and knowledge.
```
```
<!-- overview -->

This guide explains how to {insert a brief description of the task}.

{Optional: Specify when and why your user might want to perform the task.}

{Optional: Include links/tooltips to relevant concepts when terms are introduced here or in the `steps` section.}
```